test/appengine/test_urlfetch.py

```diff
@@ -0,0 +1,51 @@
+from . import MockResponse, activate_sandbox
+activate_sandbox()
```

Comment:
Maybe better to do this on suite setup and skip if the environment is not available?